internal/ingress/annotations/proxyssl/main_test.go

```diff
@@ -249,7 +249,7 @@ func TestEquals(t *testing.T) {
 
 	// Different Protocols
 	cfg1.Protocols = defaultProtocol
-	cfg2.Protocols = "SSLv3 TLSv1 TLSv1.2 TLSv1.3"
+	cfg2.Protocols = "TLSv1.2 TLSv1.3"
```

Comment:
This test case is expected to be unequal, but you modified it to be equal, so it failed.